### DISCUSSION TITLE
Here're some easy feature requests that would be amazing if we could implement.

### DISCUSSION THREAD

**Discussion by dkkb** [outline/outline#3328] [Ideas]
1.Resize the image.
#1369 #3038 

2.Support custom icons in sidecar.
Actually I've already implemented this feature, it's not complicated but it makes the sidecar colorful. It looks nice. The built-in icons are really not enough.

<img width="110" alt="image" src="https://user-images.githubusercontent.com/82504881/161379749-1e50122c-6a09-487c-b17f-aae489a5b902.png">

3.Display the outline(table of contents) in edit mode.
To be honest, the outline(table of contents) is my favorite design, it's simple but better than Notion or Evernote. It would be great to display outline in edit mode.

**Comment by tommoor** [comment]
1. Created a new issue to track: https://github.com/outline/outline/issues/3329
2. Sounds like you already implemented it – why not contribute?
3. This is already the case if you enable collaborative editing, which is the future of the project and will soon be default.
